src/Hidden/Hidden.js

```diff
@@ -1,9 +1,61 @@
 // @flow
 import React, { Element } from 'react';
 import HiddenJs from './HiddenJs';
-import type { HiddenProps } from './types';
```

Comment:
Duplicating the definitions as react-docgen isn't smart enough to resolve imports.